localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -0,0 +1,44 @@
+﻿l_english:
+ epidemic_conversion_bonus:0 ""Promote Culture" [councillor_task|E] Speed: #P +50%#!"
+
+ ##############
+ # Tribal Era #
+ ##############
+
+ innovation_amphibious:0 "Amphibious"
+ innovation_amphibious_desc:0 "We surface on land only when we please, but being natives to the water we can travel the seas at ease."
+
+ innovation_sea_faring:0 "Sea Faring"
+ innovation_sea_faring_desc:0 "Living on the coasts gave us a natural advantage in not only taming, but conquering the seas and those who dare to challenge us."
+
+ innovation_lorewalking:0 "Lorewalking"
+ innovation_lorewalking_desc:0 "Our ability to proficiently record history help us find ourselves through the lore of our ancestors."
+
+ innovation_hive_breeding:0 "Hive Breeding"
+ innovation_hive_breeding_desc:0 "Our massive hives allow us to breed countless workers and soldiers, always giving us the numerical advantage."
+ 
+ ######################
+ # Early Medieval Era #
+ ###################### 
+
+ innovation_time_is_money:0 "Time Is Money"
+ innovation_time_is_money_desc:0 "The flow if money is like the flow of time, it must be cherished, becuase without money our time here is meaningless."
```

Comment:
```suggestion
 innovation_time_is_money_desc:0 "The flow of money is like the flow of time, it must be cherished, because without money our time here is meaningless."
```